api/migrations/20210525230253-create-reminder.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("reminders", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal("uuid_generate_v4()"),
+      },
+      channel: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: {
+          model: "users",
+          key: "id",
+        },
+      },
+      body: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+      },
+      date: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
```

Comment:
Can reminders be deleted?

Reply:
Yes. Reminder will be deleted once the `date` is met, or if the user manually removes them using the `popReminder` command